src/fsharp/FSharp.Compiler-proto/FSharp.Compiler-proto.fsproj

```diff
@@ -243,12 +243,6 @@
     <Compile Include="..\..\ilx\EraseUnions.fs">
       <Link>EraseUnions.fs</Link>
     </Compile>
-    <Compile Include="..\InternalFileSystemUtils.fsi">
-      <Link>InternalFileSystemUtils.fsi</Link>
-    </Compile>
-    <Compile Include="..\InternalFileSystemUtils.fs">
-      <Link>InternalFileSystemUtils.fs</Link>
-    </Compile>
```

Comment:
Removed InternalFileSystemUtils.fs since it is better to move all of its logic into the FileSystem implementation